test/integration/react-streaming-and-server-components/test/functions.js

```diff
@@ -15,8 +15,8 @@ export default function (context) {
       'server',
       'functions-manifest.json'
     )
-    await fs.remove(join(context.appDir, '.next'))
     expect(fs.existsSync(functionsManifestPath)).toBe(false)
+    await fs.remove(join(context.appDir, '.next'))
```

Comment:
you can add another one before `nextBuild`. we need to clean the dist before `expect` checking